src/inputs/dropdown/query-search-dropdown/PQuerySearchDropdown.vue

```diff
@@ -16,12 +16,13 @@
                     <span v-for="(keyItem, idx) in querySearchState.selectedKeys" :key="idx" class="key-tag">
                         {{ keyItem.label }}:
                     </span>
-                    <input ref="inputRef" v-focus.lazy="querySearchState.isFocused"
+                    <input ref="inputRef" v-focus.lazy="querySearchState.isFocused" class="input-element"
                            :value="querySearchState.searchText"
                            :placeholder="querySearchState.currentPlaceholder || scope.placeholder"
                            :type="querySearchState.inputElType"
                            :step="querySearchState.currentDataType === 'integer' ? 1 : undefined"
                            :min="querySearchState.currentDataType === 'integer' ? 0 : undefined"
+                           size="1"
```

Comment:
👀

Reply:
The size attribute specifies the visible width, in characters, of an <input> element. (Quoted from w3schools)

Thank you for reviewing!

Reply:
As you know, type of `size` attribute would be a string.